doc/api/util.md

```diff
@@ -946,6 +946,123 @@ Otherwise, returns `false`.
+  associated string value (e.g., `node app.js --verbose`)
+  * These will be parsed automatically; you do not need to "declare" them
+  * The Flag _name_ is the string following the prefix of one-or-more dashes,
+    e.g., the name of `--foo` is `foo`
+  * Flag names become property names in the returned object
+  * When appearing _once_ in the array, the value of the property will be `true`
+  * When _repeated_ in the array, the value of the property becomes a count of
+    repetitions (e.g., `['-v', '-v' '-v']` results in `{ v: 3 }`)
+* _Options_, declared by `optionsWithValue`, which begin with one or more
+  dashes, and _do_ have an associated value (e.g., `node app.js --require
+  script.js`)
+  * Use the `optionsWithValue` option to `util.parseArgs` to declare Options
+  * The Option _name_ is the string following the prefix of one-or-more dashes,
+    e.g., the name of `--foo` is `foo`
+  * The Option _value_ is the next string following the name, e.g., the Option
+    value of `['--foo' 'bar']` is `bar`
+  * Option values may be provided _with or without_ a `=` separator (e.g.,
+    `['--require=script.js']` is equivalent to `['--require', 'script.js']`)
+  * An Option value is considered "missing" and is results in `true` when:
+    * A `=` does not separate the Option name from its value (e.g., `--foo bar`)
+      _and_ the following argument begins with a dash (e.g., `['--foo',
+      '--bar']`), _OR_
+    * The array ends with the Option name (e.g., `['--foo']`)
+  * When repeated, values are concatenated into an Array; unlike Flags, they _do
+    not_ become a numeric count
+  * When an Option name appears in the Array (or string) of `optionsWithValue`,
+    and does _not_ appear in the `argv` array, the resulting object _will not_
+    contain a property with this Option name (e.g., `util.parseArgs(['--bar'],
+    { optionsWithValue: 'foo' })` will result in `{bar: true, _: [] }`
+* _Positionals_ (or "positional arguments"), which _do not_ begin with one or
+  more dashes (e.g., `['script.js']`), _or_ every item in the `argv` Array
+  following a `--` (e.g., `['--', 'script.js']`)
+  * Positionals appear in the Array property `_` of the returned object
+  * The `_` property will _always_ be present and an Array, even if empty
+  * If present in the `argv` Array, `--` is discarded and is omitted from the
+    returned object
+  * Positionals will _always_ be parsed verbatim (e.g., `['--', '--foo']` will
+    result in an object of `{_: ['--foo']}`)
+
+A Flag or Option having the name `_` will be ignored. If it was declared as an
+Option (via the `optionsWithValue` option), its value will be ignored as well.
+
+`util.parseArgs` does not consider "short" arguments (e.g., `-v`) to be
+different than "long" arguments (e.g., `--verbose`).  Furthermore, it does not
+allow concatenation of short arguments (e.g., `-v -D` cannot be expressed as
+`-vD`).
+
+Conversion to/from "camelCase" occurs; a Flag or Option name of `no-color`
```

Comment:
Should this say "Conversion to/from "camelCase" does not occur"? It would be nice to add an example that uses camelcase flags or options.

Reply:
yes

Reply:
I've fixed that text and added further explanation